docs/api/IoT.js-API-Module.md

```diff
@@ -2,48 +2,40 @@
 
 The following shows module APIs available for each platform.
 
-|  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) |
+|  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | NuttX<br/>(STM32F4-Discovery) |
 | :---: | :---: | :---: | :---: |
 | require | O | O | O |
 
-### Contents
-
-- [Module](#module)
-    - [Module Functions](#module-functions)
-        - [`require(id)`](#requireid)
-    - [Module Loading by `require` function](#module-loading-by-require-function)
-
 # Module
-`module` implements `require` function.
+This module implements `require` function.
 Since `require` function is implicitly passed to user code, user do not need to import it explicitly.
 
-## Module Functions
-
 ### `require(id)`
-* `id <String>`- module name to be loaded
+* `id` {string} Module name to be loaded.
 
-Loads the module named 'id'
+Loads the module named `id`.
 
-# Module Loading by `require` function
+**Example**
 
-## `require` search paths
-`require` function finds modules in the order of
+```js
+var assert = require('assert');
 
-1. current working directory
-2. node_modules directory under current working directory
-3. `$HOME/node_modules`
-4. `$NODE_PATH/node_modules`
+assert.equal(2, 2);
+```
 
+**Loading a module**
 
-## `require('id')` works as follows:
-For each directory in search paths above,
+`require` function searches for modules in the following order:
 
-0. If native module named 'id' exists, load it and return.
-1. If id.js file exists, load it and return.
-2. If a directory id exists, load it as an package and return.
+1. Current working directory.
+2. `node_modules` directory under current working directory.
+3. `$HOME/node_modules`
+4. `$NODE_PATH/node_modules`
 
-## `require('id')` as a package
-For a given id, if a directory id exists, module system consider the directory as a package.
+For each directory in search paths above:
 
-0. if 'id/node_modules/package.json' contains 'main' property, load the file named main property.
-1. if either id/node_modules/package.json or main property exists, load index.js
+- If a native module named **id** exists, load it and return.
```

Comment:
I mean which module came from the IoT.js itself. (but this sentence currently exists in the document, I've just updated the style)